shared/src/commonMain/kotlin/io.newm.shared/di/NetworkClientFactory.kt

```diff
@@ -72,15 +71,17 @@ internal class NetworkClientFactory(
 
     private val refreshTokenPlugin = createClientPlugin("RefreshTokenPlugin") {
         onRequest { request, _ ->
-            if(request.url.encodedPath == "/v1/auth/refresh") {
+            if (request.url.encodedPath == "/v1/auth/refresh") {
                 request.headers.remove(HttpHeaders.Authorization)
-                request.headers.append(HttpHeaders.Authorization, "Bearer ${tokenManager.getRefreshToken()}")
+                request.headers.append(
+                    HttpHeaders.Authorization,
+                    "Bearer ${tokenManager.getRefreshToken()}"
+                )
```

Comment:
nit: I think you can just do `request.bearerAuth(tokenManager.getRefreshToken())`